### ISSUE TITLE
Comparing Different models

### ISSUE THREAD

**Issue by mohammadshahabuddin** [siyliepfl/deformation-aware-unpaired-image-translation#5]
Hi,
Could you please tell me how you compared different models? Did you use the same learning rate, number of epochs, Number of decay epochs, image size, optimizer among all models? Also, did you collect test results using the final saved generator or did you use the best results testing all saved generators at different epochs?

**Comment by siyliepfl** [comment]
Thanks. We keep the image size consistent with all models. About other hyperparameters and where to stop, for other methods such as cycleGAN, style transfer. We use the default setting in their original paper so you could refer to their implementation. For our models, we give all details in our paper.
